skp_generator/README.md

```diff
@@ -0,0 +1,56 @@
+# SKP Generator
+
+The script in this directory generates SKPs of Flutter scenes for use
+by the Skia team in their regression and performance testing.
+
+
+## Generating the SKPs
+
+Ensure that `git` and `flutter` are available on the PATH of your
+POSIX command line.
+
+Run "build.sh" to generate the SKPs. They will all be placed in the
+`skps/` directory if everything is successful.
+
+WARNING!!! THIS SCRIPT DELETES EVERYTHING NEW IN THIS DIRECTORY.
+Only files commited to Git before running the script will survive!
+
+
+## Contributing new scenes
+
+Contributions of scenes are welcome. New scenes should be
+significantly distinct from all existing scenes, and should either
```

Comment:
This will be hard to tell without Skia's help. Scenes that visually look pretty different may boil down to the same ops, and scenes that look almost exactly the same visually can boil down to very different ops.

Not sure what we can do about that, but will we be able to get feedback upstream from Skia if we're submitting substantially similar SKPs?

Reply:
I actually meant specifically different at the Flutter level (e.g. I don't want 200 scenes that all boil down to an app bar with text in the center). The point here is to have representation of typical Flutter scenes, while being agnostic about how we generate those for Skia and agnostic about how Skia handles them -- so they might generate similar SKPs today, but not tomorrow.

Reply:
Clarified to:
> Contributions of scenes are welcome. New scenes should be
> significantly distinct from existing scenes (in terms of visuals and
> Flutter widgets used to create them), and should either show a scene
> that is expensive to rasterize, or show an interesting or complex
> scene from a real-world application.

Reply:
SG